### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by hoanpl-1064**

### CODE REVIEW

**Comment by hoanpl-1064** [comment]
ready

**Comment by hoanpl-1064** [comment]
image

**Comment by hoanpl-1064** [comment]
image

**Comment by hoanpl-1064** [comment]
image

**Comment by hoanpl-1064** [comment]
image

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BookingTour</title>
```
Comment (thaodtd-0665) [inline app/views/layouts/application.html.erb]: i18n

**Comment by hoanpl-1064** [comment]
image

**Comment by hoanpl-1064** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Pull request merged**